### DISCUSSION TITLE
qutebrowser as a server browser?

### DISCUSSION THREAD

**Discussion by kd4e** [qutebrowser/qutebrowser#8148] [General] [answered]
I have Debian Server loaded for a nvr application.
I need a browser for viewing purposes and am between FF and qutebrowser.
Is there a meaningful security difference between the two, please?
I'm being told that FF has the best security of all but it seems bloated to me, which could - potentially - add vulnerabilities.
WDYT?

**Comment by The-Compiler** [comment]
Don't install a browser on a server system, easy as that. A server should not have a GUI at all.

Most security issues will be in the rendering engine rather than the UI. qutebrowser won't simplify anything there. Given that Debian Qt packages [are not upgraded to fix security issues](https://www.debian.org/releases/bookworm/amd64/release-notes/ch-information.en.html#browser-security), if you want to install from Debian packages, you'd best go for Firefox or Chromium.

**Reply by kd4e** [reply]
When at home I prefer to view the security menu and camera feeds on the dedicated server PC, via a large flat screen (non 'smart' business type) connected display, rather than being dependent on connecting an external device via the network.

This (https://qutebrowser.org/doc/install.html) is what confused me about "the underlying QtWebEngine" which is, apparently, based on Chromium Blink (https://wiki.qt.io/QtWebEngine).

**Reply by kd4e** [reply]
Follow-up ...
I think of my install like my hardware-based NVR - which has a built-in browser (or perhaps something like VLC) that enables the user to view and manipulate the images.
qutebrowser not only appears to use fewer resources, it also includes fewer potential vectors for attack vs Firefox.
WDYT?

**Reply by The-Compiler** [reply, marked as answer]
I don't really have anything more to say than what I've already said above.
